server/controllers/temporary-accommodation/manage/assessmentsController.test.ts

```diff
@@ -51,9 +51,33 @@ describe('AssessmentsController', () => {
         archivedTableRows: [],
         tableHeaders: assessmentsTableHeaders,
         archivedTableHeaders: archivedAssessmentsTableHeaders,
+        showArchived: false,
```

Comment:
I definitely see the argument for building the main view first, then adding in the archive view, but I think it any case the test wants to come in with the functionality it's testing